feat(electron): Add `BrowserWindowSession` integration (#7794)
getsentry/sentry-docs: src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx

---
title: BrowserWindowSession
excerpt: ""
description: "Captures sessions linked to the focus of Electron BrowserWindows."
sidebar_order: 15
---

> 

Reviewer: Let's add a note that this requires `4.11.0`.

Should we also talk about this affects the MainProcessSession integration?

---
title: BrowserWindowSession
excerpt: ""
description: "Captures sessions linked to the focus of Electron BrowserWindows."
sidebar_order: 15
---

Requires `@sentry/electron` version `4.11.0` or higher.

Captures sessions linked to the focus of Electron `BrowserWindow`'s. Capturing
sessions makes it possible to show [release health](/product/releases/health/)
statistics in Sentry.

> When this integration has been added, the `MainProcessSession` session

Reviewer: can we add a link to the other integration docs here?

---
title: BrowserWindowSession
excerpt: ""
description: "Captures sessions linked to the focus of Electron BrowserWindows."
sidebar_order: 15
---

Requires `@sentry/electron` version `4.11.0` or higher.

> Captures sessions linked to the focus of Electron `BrowserWindow`'s. Capturing
> sessions makes it possible to show [release health](/product/releases/health/)
> statistics in Sentry.

Reviewer 2: ```suggestion
Captures sessions linked to the focus of Electron `BrowserWindow`s. Capturing
sessions makes it possible to show [release health](/product/releases/health/)
statistics in Sentry.
```

> When this integration has been added, the <PlatformLink to="/configuration/integrations/mainprocesssession">MainProcessSession</PlatformLink>
> session integration will be disabled.

Reviewer 2: ```suggestion
Adding this integration disables the <PlatformLink to="/configuration/integrations/mainprocesssession">MainProcessSession</PlatformLink>
session integration.
```
